packages/dashboard/lib/DashboardServer.ts

```diff
@@ -70,12 +70,7 @@ export class DashboardServer {
     this.rpc = options.rpc ?? true;
     this.verbose = options.verbose ?? false;
     this.autoOpen = options.autoOpen ?? true;
-    this.frontendPath = path.join(
-      __dirname,
-      ".",
-      "dashboard-frontend",
-      "build"
-    );
+    this.frontendPath = path.join(__dirname, "dashboard-frontend");
```

Comment:
Yeah, so like I was saying on Slack - I think that when we break this out into a separate package, rather than outputting the frontend's webpack build into the server package's directory, we just refactor this line to resolve the frontend package's path via `require.resolve` instead of using a path that's relative to `__dirname`.

Reply:
Alternatively you could even have the index of the frontend module export a string array that's a list of paths to be served up by the backend/server package. Or maybe it's an object that defines a set of route keys and paths?

Either way, that allows the frontend some way to tell the backend what it contains, and allows you to add/remove/modify stuff to the frontend without changing the server logic.

Reply:
Resolving the frontend build is definitely the clean way to do this when server becomes its own package. This also means we can avoid copying in the `truffle` webpack config [here](https://github.com/trufflesuite/truffle/blob/v5.6.5/packages/truffle/webpack.config.js#L271-L282).

That list of paths to export _might_ be useful? At the moment the server listens on purpose-specific paths like `/rpc` and `/ports`, and does a [catch-all at the end](https://github.com/trufflesuite/truffle/blob/v5.6.5/packages/dashboard/lib/DashboardServer.ts#L102-L105) to let react browser router handle the rest.

Reply:
Yeah, that makes sense. Going to resolve this however as it's really just a tangent to this review.